payment-manager/src/main/java/com/qpay/paymentmanager/PaymentManagerApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
 
 @SpringBootApplication
+@EnableTransactionManagement
```

Comment:
read it carefully
https://www.baeldung.com/transaction-configuration-with-jpa-and-spring#config